README.md

```diff
@@ -4,7 +4,11 @@ Awesome Movies list, you can add your movies, like them, and reserve them.
 
```

Comment:
- [ ] Kindly add some more descriptions here about the website.
- [ ] [ OPTIONAL] KIndly change the title name that looks like the website name.